### PR TITLE
Created career model, spec and factory. Added model validations and a…

### PR DESCRIPTION
**Pull request by kvinuk**
### What does this PR do? 

 * Creates career model
 * Creates career spec
* Creates career factory
* Adds career specs to character
* Adds 1 to many relation from career to character

### CODE REVIEW

File: spec/models/career_spec.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  it { should validate_presence_of(:description) }
+  it { should validate_presence_of(:source_page) }
+  it { should validate_presence_of(:source_book) }
+  it { should validate_presence_of(:career_skills) }
+  it { should validate_presence_of(:free_ranks) }
+  it { should validate_numericality_of(:source_page).only_integer }
+  it { should validate_numericality_of(:free_ranks).only_integer }
+  it { should have_many(:characters) }
+  it "validates length of career_skills is 6" do
+  	career = FactoryGirl.create(:career)
```
Comment (kurenn) [inline spec/models/career_spec.rb]: I would make explicit the validity of the object, something like:

```ruby
career = FactoryGirl.create(:career, career_skills: ["","","","","",""])
```

File: app/models/character.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 class Character < ApplicationRecord
   belongs_to :user
   belongs_to :campaign
+  belongs_to :career
```
Comment (kurenn) [inline app/models/character.rb:4]: Please add a test for this...

**Review by kurenn — CHANGES_REQUESTED** [review]

File: app/models/character.rb (hunk 2 above)
Comment (kvinuk) [reply app/models/character.rb:4]: ponte verga

Comment (kurenn) [reply app/models/character.rb:4]: 🎉

**Review by kurenn — APPROVED** [review]

**Pull request merged**